src/Microsoft.ML.Transforms/NAHandling.cs

```diff
@@ -22,7 +21,7 @@ public static class NAHandling
         public static CommonOutputs.TransformOutput Drop(IHostEnvironment env, MissingValueDroppingTransformer.Arguments input)
         {
             var h = EntryPointUtils.CheckArgsAndCreateHost(env, MissingValueDroppingTransformer.ShortName, input);
-            var xf = new MissingValueDroppingTransformer(h, input, input.Data);
+            var xf = MissingValueDroppingTransformer.Create(h, input, input.Data);
```

Comment:
>MissingValueDroppingTransformer [](start = 21, length = 31)

please use a constructor that doesn't take input

Reply:
This is entrypoint area. I don't think we ever discussed what to do with them, so I don't want to touch them now, and I would rather have everything in consistent state, than some things are converted and some are not.

---
In reply to: [233695831](https://github.com/dotnet/machinelearning/pull/1624#discussion_r233695831) [](ancestors = 233695831)

Reply:
Yep, let's consistently avoid using `Create` methods that take data views.

---
In reply to: [233696424](https://github.com/dotnet/machinelearning/pull/1624#discussion_r233696424) [](ancestors = 233696424,233695831)